docs/references/expo/use-sso.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: '`useSSO()`'
+description: Clerk's useSSO() hook is used to create a new SSO flow.
+---
+
+The `useSSO()` hook is used to create a new SSO flow. It can be used in both web and native apps.
+
+## Parameters
+
+<Properties>
+  - `strategy`
+  - [`OAuthStrategy`](/docs/references/javascript/types/sso#o-auth-strategy)
+  - [`EnterpriseSSOStrategy`](/docs/references/javascript/types/sso#o-auth-strategy)
+
+  The strategy corresponding to the SSO provider. For example: `oauth_facebook`, `oauth_github`, `enterprise_sso`, etc.
+
+  ---
+
+  - `redirectUrl?`
+  - `string`
+
+  The URL or path to redirect to after the SSO flow is complete.
+
+  ---
+
+  - `unsafeMetadata?`
+  - [`SignUpUnsafeMetadata`](/docs/references/javascript/types/metadata#sign-up-unsafe-metadata)
+
+  Unsafe metadata to be passed to the SSO provider.
+</Properties>
+
+## Returns
+
+The `useSSO()` hook returns the `startFlow()` method, which you can use to initiate the SSO flow.
+
+The `startFlow()` method has the following signature:
+
+```ts
+const startFlow: (startSSOFlowParams?: StartSSOFlowParams) => Promise<StartSSOFlowReturnType>
+```
+
+It accepts the following parameters (`StartSSOFlowParams`):
+
+<Properties>
+  - `redirectUrl?`
+  - `string`
+
+  The URL or path to redirect to after the OAuth flow is complete.
+
+  ---
+
+  - `unsafeMetadata?`
+  - [`SignUpUnsafeMetadata`](/docs/references/javascript/types/metadata#sign-up-unsafe-metadata)
+
+  Unsafe metadata to be passed to the OAuth provider.
+
+  ---
+
+  - `identifier?`
+  - string
+
+  The [identifier](/docs/authentication/configuration/sign-up-sign-in-options#identifier) of the user.
+</Properties>
+
+## How to use the `useSSO()` hook
+
+<Include src="_partials/expo/sso-custom-flow" />
```

Comment:
Is this partial file uploaded? I can't find it.

Reply:
@victoriaxyz My bad, there was an issue with my Git history switching between branches and I've committed `useSSO` from https://github.com/clerk/clerk-docs/pull/1908

Fixed on https://github.com/clerk/clerk-docs/pull/1907/commits/d30a75cd01218addcd1bfbd823c21773f8148995